packages/shared/src/components/post/PostContent.tsx

```diff
@@ -46,7 +46,7 @@ export const ONBOARDING_OFFSET = 120;
 
 const PostContainer = classed(
   'main',
-  'flex flex-col flex-1 px-4 tablet:px-8 tablet:border-r tablet:border-theme-divider-tertiary',
+  'flex flex-col flex-1 px-4 pt-6 tablet:px-8 tablet:border-r tablet:border-theme-divider-tertiary',
```

Comment:
Can you double check it keeps modal and article page good with all the variables?
Meaning we can render that purple box yes or no,  arrows yes or no etc.